src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -385,23 +385,26 @@ static bool equalMetadataVectors(BSONObj& mdV1, BSONObj& mdV2)
 *
 * attributeValueAbsent -
 *
-* Check that the attribute doesn't have any value, taking into account its multi-valuated nature.
+* Check that the attribute doesn't have any value
 *
-* NOTE
-*   1. ValueTypeNumber and ValueTypeBoolean are always non-empty.
-*   2. In API version 2, the value of a ContextAttribute CANNOT be left out
+*/
+bool attributeValueAbsent(ContextAttribute* caP)
+{
+  return caP->valueType == ValueTypeNone;
```

Comment:
I would propose using parenthesis for returning comparisons as bool:

    return (caP->valueType == ValueTypeNone);

This is only my opinion. We need to decide what to do with this and put it is the style-guide.